docs/index.md

```diff
@@ -9,14 +9,80 @@ The [CLI](cli-reference/dub.md) can be used to
 - compile projects and external programs ([dub build](cli-reference/dub-build.md), [dub run](cli-reference/dub-run.md))
 - test projects ([dub test](cli-reference/dub-test.md))
 
-To see how to obtain dub, **go to the next page by pressing the button below**.
+DUB is bundled with most D compilers' distributions. However, it's also possible to install it separately.
+See [Installing DUB](getting-started/install.md) for details.
 
 <!-- old docs anchors for index page, all link to first steps -->
 <a id="own-projects"></a>
 <a id="adding-deps"></a>
 <a id="foreign-projects"></a>
 <a id="advanced-usage"></a>
 
-## Starting a new project
+## DUB Basics
 
-You can also skip ahead to [First steps](getting-started/first-steps.md) if you already have dub installed.
+DUB is a build tool similar to other modern languages build tools like Javascript's [npm](https://www.npmjs.com/)
+and Rust's [cargo](https://crates.io/).
+
+A file called `dub.sdl` (or `dub.json`) is used to configure a DUB project.
+
+> [SDL](https://sdlang.org/) is a "Simple Declarative Language" inspired by D's syntax.
```

Comment:
I don't think it is inspired by D's syntax though

Reply:
Yes, SDL is an independent project (possibly, but I don't remember, implemented first in a language other than D).

Reply:
Ah, I think I just guessed it :D. I will remove that comment then.

Reply:
I've changed this passage, let me know if it's ok now.